agent/agent.go

```diff
@@ -293,7 +301,10 @@ func (r *agent) LocalStatus() *pb.NodeStatus {
 	return r.recentLocalStatus()
 }
 
-type dialRPC func(*serf.Member) (*client, error)
+func (r *agent) dialMember(member *serf.Member) (*client, error) {
+	addr := fmt.Sprintf("%s:%d", member.Addr.String(), r.rpcPort)
```

Comment:
This will format as `<nil>:7676` in tests (for port == `7676`) and fail to connect as the "remote" in a test is also running on localhost.